src/Module/Profile/Photos.php

```diff
@@ -274,10 +320,11 @@ protected function post(array $request = [])
 		// Update the photo albums cache
 		Photo::clearAlbumCache($this->owner['uid']);
 
-		Hook::callAll('photo_post_end', $item_id);
-
-		// addon uploaders should call "exit()" within the photo_post_end hook
+		// addon uploaders should call "exit()" within the PHOTO_UPLOAD_END event
```

Comment:
making an own Event? `BroadCastEvent::PHOTO_UPLOAD_END_WITH_POSSIBLE_EXIT`  :D 

But tbh we should mark this event that suggesting `exit()` isn't a really bad pattern! should be replaced ...

Reply:
Ideally we should require a response object from the addons instead of calling `exit()` but this is for another PR.